test/esp/integration/alert_integration_test.rb

```diff
@@ -0,0 +1,96 @@
+require File.expand_path(File.dirname(__FILE__) + '/../../test_helper')
+
+module ESP::Integration
+  class AlertTest < ESP::Integration::TestCase
+    context ESP::Alert do
+      context 'live calls' do
+        setup do
+          @report = ESP::Report.all.detect { |r| r.status == 'complete' }
+          skip "Live DB does not have any reports.  Add a report and run tests again." if @report.blank?
```

Comment:
I think so. It will likely be a seed issue if it breaks and we won't catch it if the tests are skipped.